synapseclient/wiki.py

```diff
@@ -122,6 +122,12 @@ def __init__(self, **kwargs):
                 kwargs["attachmentFileHandleIds"].append(handle)
             del kwargs["fileHandles"]
 
+        # if parentWikiId is a falsey value (empty string, None, etc),
+        # standardize it to None
+        if "parentWikiId" in kwargs:
+            if not kwargs["parentWikiId"]:
```

Comment:
Nit: Could combine with the line above using an and statement

Reply:
Would that cause a `KeyError` if "parentWikiId" was not in `kwargs`?

Reply:
No because the second condition in an and statement is never evaluated if the first isn't True. Since the "in" condition would be False it wouldn't run into a KeyError.